src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveService.kt

```diff
@@ -110,6 +112,7 @@ class OppgaveService(
             tilordnetRessurs = tilordnetNavIdent,
             mappeId = mappeId,
             prioritet = prioritet,
+            behandlesAvApplikasjon = utledBehandlesAvApplikasjon(oppgavetype),
```

Comment:
Fungerer denne låseningen to veier? Altså at gosys ikke kan endre på oppgaver som har ts-sak som applikasjon?

Reply:
To veier er kanskje feil å si, men de skal ikke kunne avsluttes av gosys iaf, tror de fortsatt kan endes der